.stylelintrc.json

```diff
@@ -1,6 +1,482 @@
 {
   "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+    "stylelint-declaration-strict-value"
+  ],
   "rules": {
+    "order/properties-alphabetical-order": null,
+    "order/properties-order": [
+      {
+        "groupName": "all",
+        "emptyLineBefore": "always",
```

Comment:
This means that we will always need an empty line to separate this groups, so instead of this:

```css
.test {
  display: flex;
  margin: 10px 20px;
  border: 1px solid #ccc;
  padding: 16px;
}
```

we'll have this:

```css
.test {
  display: flex;

  margin: 10px 20px;

  border: 1px solid #ccc;

  padding: 16px;
}
```

This approach has some pros and cons:
Pros
- Separate nicely properties, so you can read them better
Cons
- For less properties we increase the css count lines, and this can be overkill

So I wanna know what do you think about it

Reply:
Usaria da primeira forma. Para uma classe maior isso iria ficar bem grande caso haja um espaçamento. Acredito que a maioria dos devs front estão acostumados a usar da primeira forma.

Reply:
It was discussed that we'll use the group type without an empty line between groups. Add a line to separate the groups will be responsibility of the developer when he judges necessary.